test/integration/dns_response_policy/controls/response_policy.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+control "gcp" do
```

Comment:
We prefer using the new framework for any net new tests. Could you try switching to that instead?
https://github.com/terraform-google-modules/terraform-google-network/blob/master/test/integration/simple_project/simple_project_test.go is an example

Reply:
@bharathkkb I added the golang test for dns_response_policy, auto-discovery test fail for everything except dns_managed_zone. We will have to move other tests to golang based test as well or modify the fixtures/examples a bit to make it work.